### ISSUE TITLE
Elevated circulating 4-hydroxyphenlyllactic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6428]
**Preferred term label:** Elevated circulating 4-hydroxyphenlyllactic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 4-hydroxyphenlyllactic acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating monocarboxylic acid cocentration HP:0010996

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Spelling -- 4-Hydroxyphenyllactic acid (https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_17385)

**Comment by pnrobinson** [comment]
added
